.github/workflows/release.yml

```diff
@@ -184,10 +184,8 @@ jobs:
           JIRA_SUMMARY: ${{ env.jira_summary }}
```

Comment:
These code changes involve modifying the `RELEASE_NOTES` variable in a job. The changes include adding `Ticket Details` section that includes `JIRA_SUMMARY` and `JIRA_DESCRIPTION` variables. The `RELEASE_NOTES` variable is updated to include this new section along with the existing commit details and PR description.

The code also removes the echo statements for `JIRA_SUMMARY` and `JIRA_DESCRIPTION` and modifies the `RELEASE_NOTES` variable to include these values in the final output. The `RELEASE_NOTES` variable is further processed using `sed` commands to format the text before it is used in the payload.

Overall, these changes enhance the information included in the `RELEASE_NOTES` variable by adding ticket details from JIRA and ensuring proper formatting of the text for the payload.